yt_dlp/extractor/nebula.py

```diff
@@ -253,30 +275,37 @@ class NebulaChannelIE(NebulaBaseIE):
                 'description': 'Enjoy these hottest of takes on Disney, Transformers, and Musicals.',
             },
             'playlist_mincount': 2,
+        }, {
+            'url': 'https://nebula.tv/johnnyharris',
+            'info_dict': {
+                'id': 'johnnyharris',
+                'title': 'Johnny Harris',
+                'description': 'I make videos about maps and many other things.',
+            },
+            'playlist_mincount': 90,
         },
     ]
 
     def _generate_playlist_entries(self, collection_id, channel):
-        episodes = channel['episodes']['results']
         for page_num in itertools.count(2):
-            for episode in episodes:
-                yield self._build_video_info(episode)
-            next_url = channel['episodes']['next']
+            for episode in channel['episodes']['results']:
+                metadata = self._extract_video_metadata(episode)
+                yield self.url_result(smuggle_url(
+                    episode.get('share_url') or f'https://nebula.tv/videos/{metadata["display_id"]}',
+                    {'id': metadata['id']}), NebulaIE, url_transparent=True, **metadata)
+            next_url = channel['episodes'].get('next')
             if not next_url:
                 break
-            channel = self._call_nebula_api(next_url, collection_id, auth_type='bearer',
-                                            note=f'Retrieving channel page {page_num}')
-            episodes = channel['episodes']['results']
+            channel = self._call_api(next_url, collection_id, note=f'Retrieving channel page {page_num}')
 
     def _real_extract(self, url):
         collection_id = self._match_id(url)
-        channel_url = f'https://content.watchnebula.com/video/channels/{collection_id}/'
-        channel = self._call_nebula_api(channel_url, collection_id, auth_type='bearer', note='Retrieving channel')
-        channel_details = channel['details']
+        channel = self._call_api(
+            f'https://content.watchnebula.com/video/channels/{collection_id}/',
```

Comment:
Still works, but site now uses a different endpoint:
```python
f'https://content.api.nebula.app/content/{collection_id}/?include=lessons'
```
Which has a slightly different structure and no longer includes the first set of episode results. Instead, an additional request is made to:
```python
f'https://content.api.nebula.app/video_channels/{channel['channel_id']}/video_episodes/?ordering=-published_at'
```